multisig/simulations/arbitrumsepolia/vip-011/index.ts

```diff
@@ -0,0 +1,35 @@
+import { expect } from "chai";
+import { Contract } from "ethers";
+import { ethers } from "hardhat";
+
+import { forking, pretendExecutingVip } from "../../../../src/vip-framework/index";
+import vip011, { POOL_REGISTRY, PSR } from "../../../proposals/arbitrumsepolia/vip-011";
+import PSR_ABI from "./abi/ProtocolShareReserve.json";
+
+const OLD_POOL_REGISTRY = "0x6866b2BDaaEf6648ddd5b678B3e9f3352bF3d2A5";
```

Comment:
This `PoolRegistry` was deployed with 0.8.25, wasn't it?

Implementation: https://sepolia.arbiscan.io/address/0x1b052eddd0a0072875cd3cecbf9d78e95a2aa078#code

<img width="427" alt="Screenshot 2024-06-25 at 10 31 45" src="https://github.com/VenusProtocol/vips/assets/366222/60082a0b-f865-4fdc-9a88-5eb762c85428">

Reply:
@chechu ,
I saw it now, I thought this was the reason we redeployed PoolRegistry.
I don't remind the actual reason for redeployment of IL contracts for the second time.
Here is the list which I created during deployment time, in which I deployed IL contracts three times and this address is generated from second time deployments.
https://www.notion.so/024-effcd89590694da682f78f28a59518cb?pvs=4#463871a6e6f74405b137d38005550fc7